variables.tf

```diff
@@ -49,3 +49,10 @@ variable "enable_incoming_email" {
   type        = "string"
   default     = true
 }
+
+variable "custom_spf" {
+  description = "If you use other third-party email services, you might use this to ensure you only have a single DNS SPF record"
```

Comment:
```suggestion
  description = "DNS TXT record for SPF to tell email providers which servers are allowed to send email from their domains."
```

Reply:
Again, this is not a DNS text record, it's the value portion of an SPF record.

Reply:
Also the module would then read that we are sending in spf_record rather than custom_spf. I'll agree it's not the strongest name, but spf_record seems too generic and misleading.

Reply:
Okay then maybe just add something about it being the value portion of the SPF TXT record.